identity/manager_test.go

```diff
@@ -0,0 +1,75 @@
+package identity
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/ethereum/go-ethereum/crypto"
+	"fmt"
+)
+
+func Test_CreateNewIdentity(t *testing.T) {
+	manager := NewIdentityManager("testdata")
+	id, err := manager.CreateNewIdentity()
+	assert.NoError(t, err)
+	assert.Len(t, id, 42)
+}
+
+func Test_GetIdentities(t *testing.T) {
+	manager := NewIdentityManager("testdata")
+	ids := manager.GetIdentities()
+	for _, id := range ids {
+		fmt.Println(id)
+	}
+}
+
+func Test_GetIdentity(t *testing.T) {
+	manager := NewIdentityManager("testdata")
+	ids := manager.GetIdentities()
+	for _, id := range ids {
+		identity := manager.GetIdentity(string(id))
+		assert.NotNil(t, identity)
+		assert.Equal(t, id, *identity)
+	}
+
+	identity := manager.GetIdentity("")
+	assert.Nil(t, identity)
+}
+
+func Test_HasIdentity(t *testing.T) {
+	manager := NewIdentityManager("testdata")
+	ids := manager.GetIdentities()
+	for _, id := range ids {
+		assert.True(t, manager.HasIdentity(string(id)))
+	}
+
+	identity := manager.HasIdentity("")
+	assert.False(t, identity)
+}
+
+func Test_SignMessage(t *testing.T) {
+	manager := NewIdentityManager("testdata")
+	ids := manager.GetIdentities()
+	for _, id := range ids {
+		signature, err := manager.SignMessage(id, "message to sign")
+		assert.NoError(t, err)
+		assert.Len(t, signature, 65)
+	}
+}
+func Test_SignVerifyMessage(t *testing.T) {
+
```

Comment:
`IdentityManager` knows how to sign message for an Identity, probably he should also encapsilate the `VerifyMesasgeSignature`, thought @Waldz?

Reply:
I would not commit sign/verify methods at all, because we will continue working with them later

Reply:
removed signing from this branch